network/stream/delivery.go

```diff
@@ -189,57 +176,166 @@ func (d *Delivery) Close() {
 	close(d.quit)
 }
 
-// RequestFromPeers sends a chunk retrieve request to a peer
-// The most eligible peer that hasn't already been sent to is chosen
-// TODO: define "eligible"
-func (d *Delivery) RequestFromPeers(ctx context.Context, req *network.Request) (*enode.ID, chan struct{}, error) {
-	requestFromPeersCount.Inc(1)
+// getOriginPo returns the originPo if the incoming Request has an Origin
```

Comment:
in the codebase `po` generally refers to `proximity order` which is the number of matching leading (most significant) bits between two byte slices.